Symfony/CS/Tests/Fixer/Symfony/NewlineAfterOpenTagFixerTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class NewlineAfterOpenTagFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return array(
+            array(
+                '<?php $foo = true; ?>',
+            ),
+            array(
+                '<?php $foo = true; ?>
+',
+            ),
+            array(
+                '<?php
+$foo = true;
+?>',
+            ),
+            array(
+                '<?php
+$foo = true;
+$bar = false;
+?>',
+                '<?php $foo = true;
+$bar = false;
```

Comment:
If a file contains both php and html then it is not a monolithic file and this fixer will ignore that file, is it not ?

it is some strange case of
```php
   <?php $foo = 1;
        $bar = 2;
```